### PR TITLE
mariadb 10.8.3

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by SMillerDev — DISMISSED** [review]

File: mariadb@10.7.rb [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+        release["release_number"]
+      end
+    end
+  end
+
+  keg_only :versioned_formula
+
+  # See: https://mariadb.com/kb/en/changes-improvements-in-mariadb-107/
+  deprecate! date: "2023-02-01", because: :unsupported
```
Comment (chenrui333) [inline mariadb@10.7.rb:28]: this is a short term support release.

**Comment by chenrui333** [comment]
I guess I need to move `mariadb@10.7` to the separate PR

**Comment by Bo98** [comment]
Looks like you just didn't have it in the Formula directory?

**Review by SMillerDev — APPROVED** [review]

**Comment by chenrui333** [comment]
> Looks like you just didn't have it in the Formula directory?

I see, 🤦 I will do a separate PR then.

**Pull request closed without merging**